### ISSUE TITLE
IllegalArgumentException

### ISSUE THREAD

**Issue by Nacoma** [gtache/intellij-lsp#71]
```
2019-02-26 15:14:21,834 [1932365]   WARN - tache.lsp.utils.DocumentUtils$ - LSPPOS outofbounds : Position [
  line = 174
  character = 20
] line :          column : 8 offset : 5347 
2019-02-26 15:14:21,876 [1932407]   WARN - tache.lsp.utils.DocumentUtils$ - LSPPOS outofbounds : Position [
  line = 174
  character = 12
] line :         dd column : 10 offset : 5349 
2019-02-26 15:14:21,876 [1932407]   WARN - tache.lsp.utils.DocumentUtils$ - LSPPOS outofbounds : Position [
  line = 174
  character = 14
] line :         dd column : 10 offset : 5349 
2019-02-26 15:14:21,896 [1932427]  ERROR - pplication.impl.LaterInvocator - Wrong end: 6001; document length=5994; start=5988 
java.lang.IllegalArgumentException: Wrong end: 6001; document length=5994; start=5988
	at com.intellij.openapi.editor.impl.RangeMarkerImpl.<init>(RangeMarkerImpl.java:69)
	at com.intellij.openapi.editor.impl.RangeMarkerImpl.<init>(RangeMarkerImpl.java:50)
	at com.intellij.openapi.editor.impl.RangeHighlighterImpl.<init>(RangeHighlighterImpl.java:68)
	at com.intellij.openapi.editor.impl.MarkupModelImpl.addRangeHighlighterAndChangeAttributes(MarkupModelImpl.java:115)
	at com.intellij.openapi.editor.impl.MarkupModelImpl.addRangeHighlighter(MarkupModelImpl.java:166)
	at com.github.gtache.lsp.editor.EditorEventManager.$anonfun$diagnostics$5(EditorEventManager.scala:460)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:315)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:435)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:419)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:403)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:719)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:668)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:363)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
2019-02-26 15:14:21,896 [1932427]  ERROR - pplication.impl.LaterInvocator - PhpStorm 2018.3.4  Build #PS-183.5429.47 
2019-02-26 15:14:21,896 [1932427]  ERROR - pplication.impl.LaterInvocator - JDK: 1.8.0_152-release; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o 
2019-02-26 15:14:21,896 [1932427]  ERROR - pplication.impl.LaterInvocator - OS: Linux 
2019-02-26 15:14:21,897 [1932428]  ERROR - pplication.impl.LaterInvocator - Plugin to blame: LSP Support version: 1.5.2

**Comment by gtache** [comment]
Duplicate of #70

**Comment by Nacoma** [comment]
@gtache 

I must have pasted the wrong exception. I've updated my original comment so that is shows the error mentioned in the title.

**Comment by gtache** [comment]
Okay, that should be fixed in #72, probably a timing issue.
